.github/ISSUE_TEMPLATE/03_support.md

```diff
@@ -0,0 +1,23 @@
+# 🔬 Support
+
+---
+name: 🔬 Support
+about: If you need support, check out Nock on StackOverflow
+
+---
+
+Click "Preview" for a nicer view!
+
+We primarily use GitHub as an issue tracker; for usage and support questions, please check out these resources below. Thanks! 😁.
+
+
```

Comment:
There's only one resource here. Maybe we can make this template a bit leaner, and point them to the feature and bug templates, too?

Reply:
Agree :)